pyproject.toml

```diff
@@ -33,9 +33,11 @@ dependencies = [
     "netCDF4>=1.6.4",
     "numpy",
     "nvidia-modulus >= 0.7.0",
+    "packaging",
     "pandas>=1.5.3",
     "properscoring>=0.1",
-    "pydantic >= 1.10, <=1.10.11",
+    "pydantic >= 1.10, <=3",
+    "pydantic-settings",
```

Comment:
I couldn't install pydantic<2 and pydantic-settings at the same time. Is there a way to make this installation conditional? I think it would be fair to drop support for pydantic v1.

Reply:
I thought I saw one version, I'll drop the dep and raise an exception if not present with pydantic 2

Reply:
@timkpaine That sounds good to me. Would you be able to make that change?